solutions/tests/test_is_positive.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Unit tests for the is_positive function.
+
+@author: Luyando .E. Chitindi
+"""
+
+import unittest
+from solutions.is_positive import is_positive
+
+
+class TestIsPositive(unittest.TestCase):
+    """Test the is_positive function"""
+
+    def test_positive_number(self):
```

Comment:
(Just a suggestion for conciseness, but nothing wrong here) we can be a bit more concise if we write: 
def test_positive_number(self):
   self.assertTrue(is_positive(4))
and maybe you can apply this to the two after as well. but I am not sure if my suggestion misses something from your original code.

Reply:
Hi Muna,

Thanks for the suggestion! I understand your point about conciseness, and it's actually much faster! Thank you so much.
I intentionally wrote the test cases with separate lines for actual and expected values to make the test logic clearer and easier to read.

To be honest, as I'm still getting familiar with testing and coding best practices, this format really helps me understand my own thought process and debug more efficiently. It's a bit like training wheels for my coding skills.

But you are very much right! 
I appreciate your feedback and will definitely use your concise approach in my future codes as I become more comfortable with testing.

Thank you Muna :)